utils/silversearcher-ag/Makefile

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (C) 2006-2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=silversearcher-ag
+PKG_VERSION:=0.33.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=the_silver_searcher-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://geoff.greer.fm/ag/releases/
+PKG_MD5SUM:=27aa17dbf4e0324defa341764852ba58
+
+PKG_MAINTAINER:=Nikil Mehta <nikil.mehta@gmail.com>
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/silversearcher-ag
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=very fast grep-like program, altenative to ack-grep
+  URL:=http://geoff.greer.fm/ag/
+  DEPENDS:=+liblzma +libpcre +zlib
+endef
+
+define Package/silversearcher-ag/description
+	The Silver Searcher is grep-like program implemented by C.
+	An attempt to make something better than ack-grep.
+	.
+	It searches pattern about 3–5x faster than ack-grep.
+	It ignores file patterns from your .gitignore and .hgignore.
+endef
+
+define Package/collectd/conffiles
+/etc/bash_completion.d/ag
```

Comment:
I've noticed that no other packages add bash completion, so I'll just remove this.  It's not necessary.